.env-sample

```diff
@@ -8,6 +8,11 @@ LND_CERT_BASE64='LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUNLVENDQWRDZ0F3SUJBZ0l
 # base64 ~/.lnd/data/chain/bitcoin/testnet/admin.macaroon | tr -d '\n'
 LND_MACAROON_BASE64='AgEDbG5kAvgBAwoQsyI+PK+fyb7F2UyTeZ4seRIBMBoWCgdhZGRyZXNzEgRyZWFkEgV3cml0ZRoTCgRpbmZvEgRyZWFkEgV3cml0ZRoXCghpbnZvaWNlcxIEcmVhZBIFd3JpdGUaIQoIbWFjYXJvb24SCGdlbmVyYXRlEgRyZWFkEgV3cml0ZRoWCgdtZXNzYWdlEgRyZWFkEgV3cml0ZRoXCghvZmZjaGFpbhIEcmVhZBIFd3JpdGUaFgoHb25jaGFpbhIEcmVhZBIFd3JpdGUaFAoFcGVlcnMSBHJlYWQSBXdyaXRlGhgKBnNpZ25lchIIZ2VuZXJhdGUSBHJlYWQAAAYgMt90uD6v4truTadWCjlppoeJ4hZrL1SBb09Y+4WOiI0='
 
+# Bitcoin Core Daemon RPC, used to validate addresses
+BITCOIND_RPCURL = 'http://127.0.0.1:8332'
```

Comment:
Tested! And it works like a charm. 
Needs a quick fix here to be seamless with the development environment. The default RPC port for testnet is 18332.

It rejects mainnet addresses in testnet as expected "Invalid bench32 format". Unable to connect to RPC shows the right error message. And made up strings response is "Invalid format".